composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "wordpress/plugin-check",
+    "description": "Plugin Check plugin from the WordPress Performance Team, a collection of tests to help improve plugin performance.",
+    "type": "wordpress-plugin",
+    "license": "GPL-2.0-or-later",
+    "require": {
+        "php": ">=5.6"
+    },
+    "autoload": {
+        "psr-4": {
+            "WordPress\\Plugin_Check\\": "includes/"
+        }
+    }
```

Comment:
The indentation for `.json` is set to 2 in the config: https://github.com/10up/plugin-check/blob/feature/plugin-foundation/.editorconfig#L26.

Please correct it.

Reply:
@mukeshpanchal27 this has been updated

Reply:
Thank you!